amgut/test/test_add_sample.py

```diff
@@ -116,7 +116,7 @@ def test_post_human(self):
         # Run test
         names = ag_data.ut_get_participant_names_from_ag_login_id(ag_login_id)
         response = self.post('/authed/add_sample_human/',
-                             {'participant_name': names[0],
+                             {'participant_name': names[-1],
```

Comment:
I've actually reverted this change - this was leftover from something that I was checking and I don't remember why I did that.